### PR TITLE
About

### PR DESCRIPTION
**Pull request by Damian-Bukowski**

### CODE REVIEW

File: src/main/java/com/sda/controller/CostumeController.java [hunk 1]
```diff
@@ -42,5 +42,11 @@ public String sendForm(User user) {
     public String handleCostumeForm(@ModelAttribute User user) {
         return "thank-you";
     }
+
+    @GetMapping("/about")
+    public String About() {
+        return "about";
+    }
+
```
Comment (Joanna-Adamczyk) [inline src/main/java/com/sda/controller/CostumeController.java]: zbedna linia

**Review by Joanna-Adamczyk — APPROVED** [review]

**Review by krzysztofic — APPROVED** [review]

File: src/main/java/com/sda/controller/CostumeController.java [hunk 2]
```diff
@@ -42,5 +42,10 @@ public String sendForm(User user) {
     public String handleCostumeForm(@ModelAttribute User user) {
         return "thank-you";
     }
+
+    @GetMapping("/about")
+    public String About() {
```
Comment (nowszy94) [inline src/main/java/com/sda/controller/CostumeController.java]: about z malej litery

File: src/main/resources/application.properties [hunk 3]
```diff
@@ -4,3 +4,4 @@ spring.jpa.hibernate.ddl-auto=create
 spring.datasource.url=jdbc:h2:mem:costumes
 spring.thymeleaf.mode=LEGACYHTML5
 spring.thymeleaf.cache=false
+server.port = 8081
```
Comment (nowszy94) [inline src/main/resources/application.properties]: wyrzucmy ta linie

**Review by nowszy94 — CHANGES_REQUESTED** [review]

File: src/main/java/com/sda/controller/CostumeController.java (hunk 2 above)
Comment (Damian-Bukowski) [reply src/main/java/com/sda/controller/CostumeController.java]: Resolved.

File: src/main/resources/application.properties (hunk 3 above)
Comment (Damian-Bukowski) [reply src/main/resources/application.properties]: Resolved.

**Review by nowszy94 — APPROVED** [review]

**Pull request merged**